### PR TITLE
feat: pool risks section in pool page

### PR DESCRIPTION
**Pull request by agualis**
# Description

Adds risks section to pool detail page:

- It contains the specific pool risk logic but we won't use it for now ([see comment](https://github.com/balancer/frontend-v2/pull/3359#issuecomment-1546910891) in this PR).
- It does not use i18n to simplify the logic (reduce bundle size) until we decide to translate (given that the original `Risks.vue` was not translated neither).
- We could extract hash IDs from `Risk.vue` to enforce hash correctness but we avoided it as we won't probably change them too much. 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Dependency changes
- [ ] Code refactor / cleanup
- [ ] Documentation or wording changes
- [ ] Other

## How should this be tested?

Please provide instructions so we can test. Please also list any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

## Visual context

https://github.com/balancer/frontend-v2/assets/1316240/8ceaa953-c01a-4ce5-9896-68a84e3fe948

Please provide any relevant visual context for UI changes or additions. This could be static screenshots or a loom screencast.

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have requested at least 1 review (If the PR is significant enough, use best judgement here)
- [ ] I have commented my code where relevant, particularly in hard-to-understand areas
- [ ] If package-lock.json has changes, it was intentional.
- [ ] The base of this PR is `master` if hotfix, `develop` if not

### CODE REVIEW

**Comment by agualis** [comment]
@pkattera Do we already have specific risks for a some concrete pool?

We should probably add then in a new pool metadata field @gareth?

**Comment by pkattera** [comment]
@agualis We do not have any specific risks for any pools as far as I know. I thought it may be a good idea to design for this possibility which may arise. We could just start with the category risks and add the ability to add specific risks later if the need arises if it is difficult to set up.

File: src/pages/pool/_id.vue [hunk 1]
```diff
@@ -268,6 +268,7 @@ watch(
           <template v-if="isSentinelIntersected && pool">
             <PoolTransactionsCard :pool="pool" :loading="loadingPool" />
             <PoolContractDetails :pool="pool" />
+            <PoolRisks :pool="pool"></PoolRisks>
```
Comment (garethfuller) [inline src/pages/pool/_id.vue:271]:
```suggestion
            <PoolRisks :pool="pool" />
```

**Review by garethfuller — CHANGES_REQUESTED** [review]
Because the pool risk page is so large, should we avoid using smooth scroll here? It feels a bit clumsy to click a link to a section on a page and have to wait a non-trivial amount of time for the page to scroll to the right position. wdyt @pkattera?

**Comment by garethfuller** [comment]
@pkattera I think we should get this out as is, dependent on review updates above. However, would be nice to make the risk section a little more engaging. @pkattera wdyt about having a bordered box with a relevant icon for each type of risk?

**Comment by pkattera** [comment]
> Because the pool risk page is so large, should we avoid using smooth scroll here? It feels a bit clumsy to click a link to a section on a page and have to wait a non-trivial amount of time for the page to scroll to the right position. wdyt @pkattera?

I agree that smooth scroll feels clunky when arriving from outside of the page (e.g. clicking from a pool page onto the risks page and then scrolling downwards). Ideally, the user would arrive directly at the anchor when they come from a different page. 

I do like the smooth scroll when clicking anchor links within the risks page. 

Is there an easy fix to allow both behaviors?

**Review by garethfuller — CHANGES_REQUESTED** [review]
Doesn't work on first load for me: https://www.loom.com/share/5bde7fb3128f4b65a48946dd3b6f4afc

**Comment by agualis** [comment]
> Doesn't work on first load for me: https://www.loom.com/share/5bde7fb3128f4b65a48946dd3b6f4afc

@ It should work now that I had a delay to wait for the route transition.

**Review by garethfuller — APPROVED** [review]

**Pull request merged**